component-archetype/README.md

```diff
@@ -25,7 +25,7 @@ This is a Maven archetype for creating custom Hyperdrive components.
 
```

Comment:
What about mention this in hyperdrive/README.md ? 
And commands how to switch scala version